demisto_sdk/commands/common/content/tests/objects/pack_objects/pack_metadata/pack_metadata_test.py

```diff
@@ -153,9 +153,7 @@ def test_author_getter(caplog, mocker, support, author, expected_author, expecte
     obj = PackMetaData(PACK_METADATA)
     obj.support = support
     obj.author = author
-
     assert obj.author == expected_author
```

Comment:
revert redundant changes
```suggestion
    
    assert obj.author == expected_author
    
```